src/components/Situation/Header/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as S from "./styled";
+import IconArrow from "@assets/icons/icon-arrow.svg";
+import IconHome from "@assets/icons/icon-home.svg";
+import { Link, useNavigate } from "react-router-dom";
+import PAGE_PATH from "../../../constants/path";
+
+const Header = () => {
+  const navigate = useNavigate();
+
+  const handleClickArrowIcon = () => {
+    handleGoBack();
+  };
+
+  const handleGoBack = () => {
+    navigate(-1);
+  };
+
+  return (
+    <S.Container>
+      <S.Icon src={IconArrow} onClick={handleClickArrowIcon} clickable />
```

Comment:
```suggestion
      <S.Button onClick={handleClickArrowIcon} >
        <S.Icon src={IconArrow} alt="뒤로 가기"/>
      </S.Button>
```
오호 이런 방법도 있군요 버튼으로 한번 더 감싸서 웹 접근성 살리는건 어떨까요?!
그리고 img에 alt 속성도 추가해주면 좋을 것 같아요~!